index.html

```diff
@@ -0,0 +1,874 @@
+          </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+          <section class="feature-notes col-md">
+            <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+            <div class="d-flex mt-2">
+              <h6 class="features p-1">Features</h6> <h6 class="f-category p-1 text-uppercase"><a href="#">Health</a></h6>
+            </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+          <section class="feature-notes col-md">
+            <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+            <div class="d-flex mt-2">
+              <h6 class="features p-1">Features</h6> <h6 class="f-category p-1 text-uppercase"><a   href="#">Health</a></h6>
+            </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+          <section class="feature-notes col-md">
+            <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+            <div class="d-flex mt-2">
+            <h6 class="features p-1">Features</h6> <h6 class="f-category p-1 text-uppercase"><a href="#">Health</a></h6>
+            </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+          <section class="feature-notes col-md">
+            <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+            <div class="d-flex mt-2">
+              <h6 class="features p-1">Features</h6> <h6 class="f-category p-1 text-uppercase"><a href="#">Health</a></h6>
+            </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+          <section class="feature-notes col-md">
+            <img class="img-fluid" src="img/image-1.png" alt="Image 1">
+            <div class="d-flex mt-2">
+              <h6 class="features p-1">Features</h6> <h6 class="f-category p-1 text-uppercase"><a href="#">Health</a></h6>
+            </div>
+            <h4 class="font-weight-bold"><a href="#"> Is Bootstrap for everybody?</a></h4>
+          </section>
+        </div>
+      </article>
+    <!--Editor's pick section -->
+      <article class="editor-pick bg-white p-2">
+        <div class="d-flex justify-content-center">
+          <span class="features h4 text-white pr-1 pl-1 font-weight-bold mr-2">N</span> <h2 class="text-center font-weight-bold h4 m-2">Editor's pick</h2>
+        </div>
+        <hr>
+        <div class="row row-cols-md-2">
+          <section class="travel-container col-md">
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+              <figcaption class="featured-image pl-2 pr-2 font-weight-bold">
+                Tech & Science
```

Comment:
- [ ] The badges you are using inside the images of this section should be located on the bottom-left corner of the image. Try using `position: absolute` to keep them inside the image. Check this out in the Slideshows section, too.